src/util.h

```diff
@@ -56,6 +55,8 @@ extern CTranslationInterface translationInterface;
 extern const char * const BITCOIN_CONF_FILENAME;
 extern const char * const BITCOIN_PID_FILENAME;
 
+extern std::atomic<uint32_t> logCategories;
```

Comment:
Yes being able to change the debug flags at runtime (even if not the libevent and leveldb ones which are processed at startup) would be nice.